wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/servlet/handler/FaultResponse.java

```diff
@@ -48,9 +51,19 @@ public static FaultResponse fromResponse(Response response) {
         .notSupportedResponse("Delay not supported by AEM Stubs");
     }
 
+    if (isAdminAPIIssue(response)) {
+      return FaultResponse.notSupportedResponse(response.getStatusMessage());
+    }
+
     return FaultResponse.supported();
   }
 
+  private static Boolean isAdminAPIIssue(Response response) {
```

Comment:
looks strange but I will check how it is working in practice then let you know..